src/components/gameComponents/individualGameScreenComponents/GameStockSearchResults.js

```diff
@@ -0,0 +1,119 @@
+import { Row, Col, Container, Button } from 'react-bootstrap';
+import { APIName } from '../../../constants/APIConstants';
+import { useState, useEffect } from 'react';
+import { API } from 'aws-amplify';
+import { useSelector } from 'react-redux';
+import LoadingSpinner from '../../widgets/LoadingSpinner/LoadingSpinner';
+import MessageAlert from '../../widgets/MessageAlert/MessageAlert';
+import TickerCard from '../../stockDiscoveryComponents/tickercard/Tickercard';
+
+function GameStockSearchResults({ keyword, league }) {
+  const [stocks, setStock] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState('');
+  const [page, setPage] = useState(0);
+
+  const user = useSelector((state) => state.user);
+  const { userInfo } = user;
+  const userToken = userInfo.token;
+
+  function next() {
+    if (!error) {
+      setPage(page + 1);
+    }
+  }
+  function prev() {
+    if (page > 0) {
+      setPage(page - 1);
+    }
+  }
+
+  useEffect(() => {
+    const getStocks = async () => {
+      try {
+        setLoading(true);
+        let path = `/api/stock/gameStocks/search/${keyword}`;
+        let body = {
```

Comment:
Pass these as query parmas instead of body, have a look [here](https://docs.amplify.aws/lib/restapi/fetch/q/platform/js/)